buildpipeline/DotNet-CoreFx-Trusted-Windows-UWP.json

```diff
@@ -0,0 +1,487 @@
+{
+  "build": [
```

Comment:
Just for consistency can you call this UAP instead of UWP. I know they are generally considered the same but I prefer the consistency.